### PR TITLE
New/controller

### PR DESCRIPTION
**Pull request by ratladi**
Please review our code on creating a hello world controller

### CODE REVIEW

**Comment by ahmeed83** [comment]
I agree with leandrotula

File: todoapp.iml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <orderEntry type="library" name="Maven: org.springframework.boot:spring-boot:2.2.5.RELEASE" level="project" />
+    <orderEntry type="library" name="Maven: org.springframework.boot:spring-boot-autoconfigure:2.2.5.RELEASE" level="project" />
+    <orderEntry type="library" name="Maven: org.springframework.boot:spring-boot-starter-logging:2.2.5.RELEASE" level="project" />
+    <orderEntry type="library" name="Maven: ch.qos.logback:logback-classic:1.2.3" level="project" />
+    <orderEntry type="library" name="Maven: ch.qos.logback:logback-core:1.2.3" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.logging.log4j:log4j-to-slf4j:2.12.1" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.logging.log4j:log4j-api:2.12.1" level="project" />
+    <orderEntry type="library" name="Maven: org.slf4j:jul-to-slf4j:1.7.30" level="project" />
+    <orderEntry type="library" name="Maven: jakarta.annotation:jakarta.annotation-api:1.3.5" level="project" />
+    <orderEntry type="library" scope="RUNTIME" name="Maven: org.yaml:snakeyaml:1.25" level="project" />
```
Comment (aniwange33) [inline todoapp.iml]: make sure git ignore these files
.idea
/workspace
/target
/todoapp.iml

**Comment by ratladi** [comment]
Thanks guys, will work on it with the group

File: todoapp.iml (hunk 1 above)
Comment (ratladi) [reply todoapp.iml]: made the changes

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```
Comment (ahmeed83) [inline .idea/.gitignore]: we dont need this file. delete it

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
```
Comment (ahmeed83) [inline .gitignore]: looks good

File: .idea/codeStyles/codeStyleConfig.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
```
Comment (ahmeed83) [inline .idea/codeStyles/codeStyleConfig.xml]: don't add .idea files to git. These are your files

File: .idea/compiler.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <module name="todoapp" />
+      </profile>
+    </annotationProcessing>
+    <bytecodeTargetLevel>
+      <module name="todoapp" target="1.8" />
+    </bytecodeTargetLevel>
+  </component>
+</project>
```
Comment (ahmeed83) [inline .idea/compiler.xml]: don't add .idea files to git. These are your files

**Pull request closed without merging**